routers/routes/routes.go

```diff
@@ -608,7 +608,6 @@ func RegisterRoutes(m *macaron.Macaron) {
 
 		m.Group("/wiki", func() {
 			m.Get("/raw/*", repo.WikiRaw)
-			m.Get("/*", repo.WikiRaw)
```

Comment:
Why this route is removed?

Reply:
It cannot be reached (aside from a few corner cases), since the `/:page` route takes precedence.